modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -313,6 +305,33 @@ def __parse_output(raw_output):
     def __alives(self):
         return [node for node in self._cluster.nodes if self._cluster.alive(node)]
 
+    def _parse_ssl_params(self, user, globals_dict, **kwargs):
+        ssl_dict = None
+        if globals_dict.get('use_ssl', False):
+            ssl_dict = globals_dict.get(user, {}).get('ssl', {})
+        elif kwargs.get('key_store_jks') is not None or kwargs.get('key_store_path') is not None:
+            ssl_dict = kwargs
+        return None if ssl_dict is None else \
+            SslContextFactory(key_store_path=ssl_dict.get("key_store_path",
```

Comment:
I suppose,that some of these lines of code can be extracted to method.
You can define it in __parse_ssl_params.

```
def get_store_path(type, ssl_dict):
    path_key = f'{type}_path'
    store_name = f'{type}_jks'
    default_name = DEFAULT_TRUSTSTORE if type == 'trust_store' else DEFAULT_ADMIN_KEYSTORE
    return ssl_dict.get(path_key, self.jks_path(ssl_disct.get(store_name, default_name))